mixer/adapter/list/list.go

```diff
@@ -79,7 +80,28 @@ func (h *handler) HandleListEntry(_ context.Context, entry *listentry.Instance)
 		return adapter.CheckResult{}, err
 	}
 
-	found, err := l.checkList(entry.Value)
+	var value string
+	isString := true
+
+	switch entryVal := entry.Value.(type) {
```

Comment:
Wouldn't it be simpler to always normalize the input into a string form? Then you don't need to change the ipList.go code at all.

Reply:
ipList.go code needs to change because the function to convert to ip between byte array and string is different. I tried to just use string representation of byte array but then net.ParseIP(<string ip>) function didn't return right IP address...

Reply:
What I mean is that you could convert the incoming byte[] into an IP address string in this code, so then checkList would always just get a string and convert it to an IP address object.